spawn:
    # grid size in meters:
    grid_size: 2.0